spec/src/main/asciidoc/configsources.asciidoc

```diff
@@ -48,9 +46,10 @@ com.acme.myproject.someserver.url = http://more_important.server/some/endpoint
 
 A Microprofile-Config implementation must provide <<ConfigSource,ConfigSources>> for the following data out of the box:
 
-* System properties (ordinal=400)
-* Environment variables (ordinal=300).  
-* A `ConfigSource` for each property file `META-INF/microprofile-config.properties` found on the classpath. (default ordinal = 100)
+* System properties (ordinal=400).
```

Comment:
In System properties and env vars, this should be changed to "(default ordinal = 400)" as with `config_value` the ordinal can be changed also for these config sources.